examples/gradle-and-jvm-no-json/.gitignore

```diff
@@ -0,0 +1,3 @@
+gradle/*
+gradlew
+gradlew.bat
```

Comment:
This lets me open the project in intelij and not have to check these files in.

Reply:
👍  Can you add this to `examples/.gitignore` instead, since we'll have the same problem with all of the example projects?